corehq/apps/hqcase/management/commands/ptop_preindex.py

```diff
@@ -21,8 +21,14 @@ def get_reindex_commands(alias_name):
     # that should be used to rebuild the index from scratch
     pillow_command_map = {
         'hqdomains': [('ptop_reindexer_v2', {'index': 'domain'})],
-        'hqcases': ['ptop_fast_reindex_cases'],
-        'xforms': ['ptop_fast_reindex_xforms'],
+        'hqcases': [
+            'ptop_fast_reindex_cases',
+            ('ptop_reindexer_v2', {'index': 'sql-case'}),
```

Comment:
grabbed the name from https://github.com/dimagi/commcare-hq/blob/0db3decb4400d8b7321cf4daac485a2448f463ba/corehq/apps/hqcase/management/commands/ptop_reindexer_v2.py#L41